gleam.toml

```diff
@@ -7,7 +7,7 @@ repository = { type = "github", user = "grottohub", repo = "glyph" }
 links = [{ title = "Website", href = "https://gleam.run" }]
 
 target = "erlang"
-gleam = ">= 1.0.0"
+gleam = ">= 0.32.0"
```

Comment:
is there a reason we've lowered the minimum version?

Reply:
Just mistake